apps/angular/src/app/app.component.html

```diff
@@ -1 +1,3 @@
-<router-outlet></router-outlet>
+<div class="container">
+	<router-outlet></router-outlet>
```

Comment:
You can use self-closing tags.
```suggestion
	<router-outlet/>
```